src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
Apparently, you forgot to add this class to WebAppInitializer.
![image](https://user-images.githubusercontent.com/63553735/206858176-6c7e6041-1f5a-4d2e-9066-8950e1dacbe0.png)